core/modules/order/store/mutations.ts

```diff
@@ -30,6 +30,12 @@ const mutations: MutationTree<OrderState> = {
   },
   [types.ORDER_LAST_ORDER_WITH_CONFIRMATION] (state, payload) {
     state.last_order_confirmation = payload
+  },
+  [types.ORDER_ADD_SESSION_ORDER_HASH] (state, hash: string) {
+    state.session_order_hashes.push(hash)
+  },
+  [types.ORDER_REMOVE_SESSION_ORDER_HASH] (state, hash: string) {
+    state.session_order_hashes = state.session_order_hashes.filter(sessionHash => sessionHash !== hash)
```

Comment:
@patzick shouldn't we store it localStorage (plugin for syncing state with localStorage - sth like a shopping cart)? I I mean - I'm not sure. Most cases are covered just by this approach and its' totally fine with me

Reply:
i think we shouldn't - it's a check for multiple order invocation and works realtime.
Aaand about this localStorage....when you place an order without registration and go to homepage, even refresh page, go place another order and...user data filled in checkout. I think user data shouldn't be stored there.